packages/cryptoplease/lib/app/components/usdc_info.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:cryptoplease/l10n/l10n.dart';
+import 'package:cryptoplease/ui/info_icon.dart';
+import 'package:cryptoplease/ui/info_widget.dart';
+import 'package:flutter/material.dart';
+
+class UsdcInfoWidget extends StatelessWidget {
+  const UsdcInfoWidget({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final screenHeight = MediaQuery.of(context).size.height;
```

Comment:
This was true for the specific screen, if you're making this widget reusable, it should rather use `LayoutBuilder` and calculate the sizes based on the available layout constraints, not on the screen size.